src/js/view/view.js

```diff
@@ -940,7 +953,10 @@ function View(_api, _model) {
                 top: null,
                 bottom: null
             });
-            _this.floatingUI.destroy();
+            if (_this.floatingUI) {
```

Comment:
you're duplicating this functionality in two places, since it uses the same locally scoped var we can abstract it out in the event we need to update it in the future. also ensures that any future changes to this conditional only require 1 update